### ISSUE TITLE
Visualization

### ISSUE THREAD

**Issue by zjx99** [sshaoshuai/PointRCNN#43]
Hi, shaoshuai,
I am wondering how to show the detection results with 3D bbxes and show the detected vehicles in a distinguish color.  Did you use any library?  Thanks.

**Comment by sshaoshuai** [comment]
Hi， I use `mayavi` for all the visualization of point cloud, which is a python library.

**Comment by zjx99** [comment]
Hi, shaoshuai,
You pointed out that this work takes lidar as the only input, so I am quite confused about images included in testing folder. What are they used for in testing stage?  If they are not used in the testing scheme itself, then it should work when I put my own lidar data which has been transfered into bin files, is that right?   Thanks

**Comment by sshaoshuai** [comment]
Yes, the code just uses the image_shape to crop the lidar points within the image. You could delete related codes to apply on the full areas of lidar scenes in your dataset.

**Comment by zjx99** [comment]
Hi, 
I am still struggling with showing 3d boxes on lidar points directly.  
Here is an example in the output file: 
Car -1 -1 -1.7723 753.6551 163.8756 814.0421 204.0525 1.5462 (height) 1.6426 (width) 3.9845 (length) **7.0502 (x) 1.2010 (y) 29.7814 (z)** -1.5398 (theta) 1.6346 
As (x,y,z) is in camera coordinate system, I have no idea how to transfer it to Velodyne coordinate. Is there any transfermation matrices needed? 
Also, why is the first nubmer behind "car" "-1" rather than float from 0 to 1 ? 

Thanks

**Comment by iris0329** [comment]
Hi @zjx99, 
do you know what  `-1 -1 -1.7723 753.6551 163.8756 814.0421 204.0525 ` means ?
Thanks  !

**Comment by iris0329** [comment]
You can refer to visual code of frustum-pointnet .

https://github.com/sshaoshuai/PointRCNN/issues/16#issuecomment-507593480

**Comment by chenxyyy** [comment]
You can refer to visual code here
```https://github.com/kuixu/kitti_object_vis.git```
